js/models/media_overlay.js

```diff
@@ -59,6 +77,17 @@ var MediaOverlay = function(package) {
         return undefined;
     };
     
+    /**
+     * Calculates a timecode corresponding to a percent of the total audio duration
+     * Note: never called; no returned value; obsolete?
```

Comment:
This is used here: https://github.com/readium/readium-js-viewer/blob/develop/src/js/EpubReaderMediaOverlays.js#L420